packages/flutter/test/material/text_field_test.dart

```diff
@@ -1935,6 +1897,7 @@ void main() {
     final Offset slightRight = iPos + const Offset(30.0, 0.0);
     await tester.longPressAt(slightRight, pointer: 7);
     await tester.pump();
+    await tester.pump();
```

Comment:
Is this still needed?

Reply:
No, I thought I removed all of these!  Thanks.